exporter/stackdriver/stats.go

```diff
@@ -264,17 +263,17 @@ func (e *statsExporter) createMeasure(ctx context.Context, vd *view.Data) error
 	var metricKind metricpb.MetricDescriptor_MetricKind
 	var valueType metricpb.MetricDescriptor_ValueType
 
-	switch agg.Type {
-	case view.AggTypeCount:
+	switch agg {
+	case exporter.AggTypeCount:
 		valueType = metricpb.MetricDescriptor_INT64
-	case view.AggTypeSum:
+	case exporter.AggTypeSum:
 		valueType = metricpb.MetricDescriptor_DOUBLE
-	case view.AggTypeMean:
+	case exporter.AggTypeMean:
 		valueType = metricpb.MetricDescriptor_DISTRIBUTION
-	case view.AggTypeDistribution:
+	case exporter.AggTypeDistribution:
 		valueType = metricpb.MetricDescriptor_DISTRIBUTION
 	default:
-		return fmt.Errorf("unsupported aggregation type: %s", agg.Type.String())
+		return fmt.Errorf("unsupported aggregation type: %s", agg.String())
```

Comment:
`.String()` automatically be invoked by the "%s" format specifier https://golang.org/pkg/fmt/#hdr-Printing